addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -346,13 +346,14 @@ PortalChatter.include({
         var messageIndex = $source.data("mes_index");
 
         var comment = this.messages[messageIndex].rating.publisher_comment;
-        this._getCommentContainer($source).empty();
         if (comment) {
             var data = {
                 rating: this.messages[messageIndex].rating,
                 is_publisher: this.options.is_user_publisher,
             };
-            this._getCommentContainer($source).append(renderToElement("portal_rating.chatter_rating_publisher_comment", data));
+            this._getCommentContainer($source).empty().append(renderToElement("portal_rating.chatter_rating_publisher_comment", data));
+        } else {
+            this._getCommentContainer($source).empty();
         }
```

Comment:
Ah yes I see, as _getCommentContainer is relative, when emptying it beforehand, it does not retrieve the element. LGTM :)